addon/-private/data-view/radar/dynamic-radar.js

```diff
@@ -133,14 +136,14 @@ export default class DynamicRadar extends Radar {
       const {
         top: currentItemTop,
         height: currentItemHeight
-      } = currentItem.getBoundingClientRect();
+      } = getScaledClientRect(currentItem, _transformScale);
 
       let margin;
 
       if (previousItem !== undefined) {
-        margin = currentItemTop - previousItem.getBoundingClientRect().bottom;
+        margin = currentItemTop - getScaledClientRect(previousItem, _transformScale).bottom;
       } else {
-        margin = currentItemTop - itemContainer.getBoundingClientRect().top - totalBefore;
+        margin = currentItemTop - getScaledClientRect(itemContainer, _transformScale).top - totalBefore;
```

Comment:
we seem to be duplicating work here

Reply:
We won't be, this case will only happen on the very first item. I can add a comment to make this more explicit

Reply:
I'm not sure I follow,  it seems like you are generating the boundingClientRect twice

Reply:
Nvm I'm dense carry on